site/static/compare.html

```diff
@@ -46,18 +47,20 @@ <h3>Cache states</h3>
         </ul>
     </div>
     <div id="content" style="display: none"></div>
-    <div id="settings">
-        <div id="commits" class="settings" style="text-align:left;">
-            <h3>Commits</h3>
-            Commit/Date A: <input width="100em" placeholder="YYYY-MM-DD or SHA" id="start-bound" /><br>
-            Commit/Date B: <input width="100em" placeholder="YYYY-MM-DD or SHA" id="end-bound" /><br>
-            <select id='stats' name="stat">
-            </select>
-            <div class="submit">
-                <a href="#" onClick="submit_settings(); return false;">Submit</a>
-            </div>
-        </div>
-    </div>
+    <form id="settings" action="">
+        <fieldset id="commits">
+            <legend>Commits</legend>
```

Comment:
Maybe you could adjust the legend to eg "Select comparision" ~~or "Choose commits to compare"~~ ?